sdk/storagesync/Microsoft.Azure.Management.StorageSync/Tools/run_devshell.cmd

```diff
@@ -0,0 +1,28 @@
+@if not "%DevShellPath%"=="" @goto :devshellpath_ok
+
+@echo ERROR: Environment variable DevShellPath is not set.
+@exit /b 1
+
+:devshellpath_ok
+
+@if exist "%DevShellPath%\devshell.psm1" @goto :devshellcmd_ok
+
+@echo ERROR: devshell.psm1 is not found at DevShellPath=%DevShellPath%
+@exit /b 1
+
+:devshellcmd_ok
+
+@REM This customization adds an external library to DevShell.
+@REM That would display commands from the module in DevShell's "info"
+@REM Note that DevShell currently hardcodes each library it operates,
+@REM so reloading devshell with "init" will lose our custom library.
+@set DevShellCustomizations=ipmo %~dp0\Utils.psm1 -DisableNameChecking; Set-LibraryInfo -LibraryName Utils -LibraryVersion '1.0' -LibraryDisplayName StorageSyncSdkUtils -LibraryDescription StorageSyncSdkDevOps;
+@set DevShellLoader=%DevShellPath%\devshell.psm1
+@set PowerShellLoader=%windir%\system32\WindowsPowerShell\v1.0\powershell.exe
```

Comment:
Will be great if you provide an explanation for such code.
Is this meant for only windows user? Will this create any issue for non-windows user?

Reply:
Hi Abhijet, these are the devops utilities we maintain in our AZ Cmdlets repository and I am bringing them to our SDK repo as well since they provide convenience for tedious tasks of configuring/building/recording/replaying, etc.

Reply:
what happens there is that devshell (out nuget package with devops utilities) is being extended with with one more module, utils.psm1 - now anybody in our team can implement common tasks in utils.psm1 using common functionality provided by the nuget - and anybody else in the team will benefit from it. Utilities can be anything really, currently I am using keyvault to store test environment secrets as as values of TEST_CSM_ORGID_AUTHENTICATION

Reply:
> Is this meant for only windows user? Will this create any issue for non-windows user?

it is meant for developers using Windows and since we don't have other developers, no issue here.
It has no effect on users.